apps/website/src/hooks/consent.tsx

```diff
@@ -127,7 +127,13 @@ const ConsentDialog: React.FC<{ context: ConsentContext }> = ({ context }) => {
 
     // If the user hasn't interacted, and this is the first run, show the dialog
     // But, don't show it if the user is on the privacy page
-    if (hasInteracted === null && router.pathname !== "/privacy-policy") {
+    // And, don't show it if the user has a GPC/DNT signal
+    if (
+      hasInteracted === null &&
+      router.pathname !== "/privacy-policy" &&
+      navigator.doNotTrack !== "1" &&
+      !navigator.globalPrivacyControl
+    ) {
```

Comment:
Lmk if you disagree with this being the experience for GPC/DNT -- I didn't want to explicitly write the opt-out to local storage, so I just left them in the "hasn't interacted" bucket (so if they ever stop sending the signals, they'll get the usual "welcome" consent popup)

Reply:
no that's perfect